app/assets/stylesheets/refinery/find_out.scss

```diff
@@ -50,7 +50,10 @@
       background: none;
     }
 
-    background-image: image-url('group-2111.png');
+    background-image: image-url('376-find-out-banner.png');
+    background-position: center center;
+    background-size: contain;
```

Comment:
Did you also consider `background-size: cover` for this? I notice at lower sizes the background image shrinks on both axes and wonder if you think it would work better?

<img width="962" alt="image" src="https://user-images.githubusercontent.com/565647/62730959-16104c80-b9ef-11e9-8f2d-b75b1b89da5f.png">

versus

<img width="946" alt="image" src="https://user-images.githubusercontent.com/565647/62730983-24f6ff00-b9ef-11e9-8b28-3ebc14c4ba5d.png">

Reply:
I had not considered `cover` in lieu of `contain` and agree that it looks neater. I'll update that as well in the commit.